<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormMarchandise</class>
 <widget class="QDialog" name="FormMarchandise">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>831</width>
    <height>571</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ajouter Marchandise</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#FormMarchandise {
    border-image: url(&quot;C:/projet_QT/projet_achats/image000/fond02.jpg&quot;) 0 0 0 0 stretch stretch;
}


</string>
  </property>
  <widget class="QLineEdit" name="lineEditRef">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>170</y>
     <width>301</width>
     <height>30</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>RÃ©f Marchandise</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditDesign">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>290</y>
     <width>331</width>
     <height>30</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>DÃ©signation</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPrix">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>230</y>
     <width>251</width>
     <height>30</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Prix de Vente</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAjouter">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>480</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: #29bf12;
border-radius: 15px;
border: none;
padding: 10px 20px;
color: white;
font-size: 15px;
font-weight: bold;
}</string>
   </property>
   <property name="text">
    <string>ðŸ“¥ Enregistrer</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-size: 15px;
    font-weight: bold;
    color: white;
}
</string>
   </property>
   <property name="text">
    <string>RÃ©f Marchandise:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-size: 15px;
    font-weight: bold;
    color: white;
}
</string>
   </property>
   <property name="text">
    <string>Prix de Vente:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>290</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-size: 15px;
    font-weight: bold;
    color: white;
}
</string>
   </property>
   <property name="text">
    <string>DÃ©signation:</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>811</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextBrowser {
    border-image: url(&quot;C:/projet_QT/projet_achats/image000/fond03.jpg&quot;) 0 0 0 0 stretch stretch;
    border: none;
    padding: 10px 20px;
    color: white;
    font-weight: bold;
}</string>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:700; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:20pt;&quot;&gt;Nouvelle marchandise :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
